src/jsifier.js

```diff
@@ -432,7 +432,10 @@ function JSify(data, functionsOnly) {
           print('STATIC_BASE = GLOBAL_BASE;\n');
           print('STATICTOP = STATIC_BASE + ' + Runtime.alignMemory(Variables.nextIndexedOffset) + ';\n');
         } else {
-          print('gb = alignMemory(getMemory({{{ STATIC_BUMP }}}, ' + MAX_GLOBAL_ALIGN + ' || 1));\n');
+          print('gb = alignMemory(getMemory({{{ STATIC_BUMP }}}), ' + MAX_GLOBAL_ALIGN + ' || 1);\n');
+          // Ideally, for efficiency, this would only zero memory after the static initialized
+          // data, but we don't know the size of that here.
+          print('for (let i = gb; i < gb + {{{ STATIC_BUMP }}}; ++i) HEAP8[i] = 0;\n');
```

Comment:
while I agree `let` is nicer, please use `var` for consistency with the rest of the code. (might be issues with older VMs, too)